content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+it launches or that are explicitly launched by the user, are all assigned to the
+job object of that container. When the init process exits or is signaled to exit,
+all the processes in the job will be signaled to exit, the job handle will be
+closed and the storage will be unmounted.
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own pod security policies. Those used to configure Linux privileged
+containers **do not** apply. Enabling privileged access to a Windows host is a
+fundamentally different process than with Linux so the configuration and
+capabilities of each differ significantly. Below is a diagram detailing the
+overall architecture of Windows HostProcess containers:
+
+{{< figure src="hpc-architecture.png" alt="HostProcess Architecture" >}}
+
+Two major features were added prior to moving to stable: the ability to run as local user accounts, and
+a simplified method of accessing volume mounts. For more info visit our [updated documentation page](/docs/tasks/configure-pod-container/create-hostprocess-pod)
+
+## Use Cases
+
+The SIG-Windows community has been busy putting HostProcess containers to use - even before GA! CThey've
+been very excited to use HostProcess containers for a number of important activities that were a pain to
+perform in the past. Now using HostProcess containers you can install the windows-exporter on your nodes
+and collect extremely detailed monitoring information with Prometheus. You can now run csi-proxy, flannel,
+calico, event loggers, and more as HPCs.
+
+To see more HPC examples head over to [HostProcess Examples](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess)
+
+## How do I use it?
+
+A HostProcess container can be built using any base image of your choosing, however, for convenience we have
+created [a HostProcess container base image](https://github.com/microsoft/windows-host-process-containers-base-image).
+This image is only a few KB in size and does not inherit any of the same compatibility requirements as regular Windows
+server containers.
+
+All you need in your Dockerfile is the following URI:
+
+```dockerfile
+FROM mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0
+```
+
+HostProcess containers can be run from within a
+[HostProcess Pod](/docs/tasks/configure-pod-container/create-hostprocess-pod).
+With the feature enabled on Kubernetes version 1.22, a containerd container runtime of
+1.7 or higher, and the latest version of hcsshim, deploying a pod spec with the
+[correct HostProcess configuration](/docs/tasks/configure-pod-container/create-hostprocess-pod/#before-you-begin)
+will enable you to run HostProcess containers. To get started with running
+Windows containers see the general guidance for [Windows in Kubernetes](/docs/setup/production-environment/windows/)
```

Comment:
```suggestion
To get started with running Windows containers,
see the general guidance for [deploying Windows nodes](/docs/setup/production-environment/windows/). 
If you have a compatible node (for example: Windows as the operating system
containerd 1.7 or later, and hcsshim v0.9.5), you can deploy a Pod with one
or more HostProcess containers.
See the [prerequisites](/docs/tasks/configure-pod-container/create-hostprocess-pod/#before-you-begin)
in [Create a Windows HostProcess Pod](/docs/tasks/configure-pod-container/create-hostprocess-pod)
for more information.
Within a Pod, you can't mix HostProcess containers with normal Windows containers.
```

Reply:
I incorporated this suggestion in manually (I couldn't do it through the website)